src/base.iced

```diff
@@ -178,23 +178,38 @@ class Base
 
   #------
 
-  _v_include_pgp_hash : -> false
-  _v_require_full_pgp_hash : -> false
-  _v_pgp_km_to_hash : () -> @km()
-  _v_pgp_hash_dest : (body) -> body.key
+  _v_include_pgp_details : -> false
+  _v_require_pgp_details : -> false
+  _v_pgp_details_dest : (body) -> body.key
```

Comment:
That is, can we make this function into whatever our preferred approximation of "pure virtual" is?